react/fields/MonthPicker/MonthPicker.js

```diff
@@ -0,0 +1,175 @@
+import styles from './MonthPicker.less';
+
+import React, { Component, PropTypes } from 'react';
+import classnames from 'classnames';
+
+import ErrorIcon from '../../icons/ErrorIcon/ErrorIcon';
+import CustomMonthPicker from './CustomMonthPicker/CustomMonthPicker';
+import NativeMonthPicker from './NativeMonthPicker/NativeMonthPicker';
+
+function combineClassNames(props = {}, ...classNames) {
+  const { className, ...restProps } = props;
+
+  return {
+    className: classnames.apply(null, [...classNames, className]), // eslint-disable-line no-useless-call
+    ...restProps
+  };
+}
+
+export default class MonthPicker extends Component {
+
+  static displayName = 'MonthPicker';
+
+  static propTypes = {
+    /* eslint-disable consistent-return */
+    id: (props, propName, componentName) => {
```

Comment:
I noticed that `Dropdown` uses the same propType validator. I wonder if it's worth extracting these into a shared file that contains common `PropTypes`.